tests/functional/ext/typing/typing_broken_noreturn.py

```diff
@@ -0,0 +1,38 @@
+"""
+'typing.NoReturn' is broken inside compond types for Python 3.7.0
+https://bugs.python.org/issue34921
+
+If no runtime introspection is required, use string annotations instead.
+"""
+# pylint: disable=missing-docstring
+import sys
+import typing
+from typing import Callable, Union
+
+import typing_extensions
+
+if sys.version_info >= (3, 6, 2):
```

Comment:
Is this still necessary for our tests? According to the `.rc` file we don't even run on `3.6`?

Reply:
We're incompatible with python 3.6.2 since pylint 2.12.1 (https://github.com/PyCQA/pylint/pull/5068) so I think we can remove all those chekck : >= 3.6.2 it's always true.

Reply:
My bad I did not see it was in a test, my comment is irrelevant.

Reply:
> My bad I did not see it was in a test, my comment is irrelevant.

However even the tests require `3.6.2`. So I think the initial comment is still valid.